### ISSUE TITLE
Will only sync to 1(of 2) outlook PC, whichever initiates first, 2-way

### ISSUE THREAD

**Issue by BanyonLabs** [phw198/OutlookGoogleCalendarSync#292]
I am having the issue where it will mirror naturally on an android phone from for the calendar and to whatever computer syncs first out of two total computers. The second computer that attempts to sync will not pick up any changes made by the first computer, leaving events off the calendar for that second device. This happens whether the computer made the event or it was directly from gmail/phone. 

All settings exact, all versions and OS exact. 

The same goes for if the second device creates or syncs first. It will not pick up on the first computer (though fine on gmail/phone),

Am I missing something obvious here, is it designed to sync up to only 1 outlook location?

TLDR
Gmail / Phone  mirrored -no issues
2 Desktops - whichever starts auto-sync first two-way will be updated. Other updates "successfully" but no changes to second Desktop.

Thank you.

**Comment by phw198** [comment]
It sounds like you are syncing to two different computers with Outlook, to a central, common Google calendar. Unfortunately, that's not yet a supported configuration (3rd from the bottom).
<img src="https://github.com/phw198/OutlookGoogleCalendarSync/raw/master/docs/images/supported_sync_configs.png" width="200"/>
